contract/Cargo.toml

```diff
@@ -1,19 +1,25 @@
 [package]
 name = "contract"
-version = "0.1.1"
-authors = ["Maciej Zielinski <maciej@casperlabs.io>"]
+version = "0.1.0"
+authors = ["zie1ony <maciej@casperlabs.io>"]
```

Comment:
I prefer to have `Maciej Zielinski` here.

Reply:
Renamed, sorry for that